src/controllers/lyrics.js

```diff
@@ -0,0 +1,238 @@
+            itemsContainer.innerHTML = html;
+        }
+
+        const lyricLineArray = itemsContainer.querySelectorAll('.lyrics');
+
+        // attaches click event listener to change playtime to lyric start
+        lyricLineArray.forEach(element => {
+            element.addEventListener('click', () => onLyricClick(element.getAttribute('data-lyrictime')));
+        });
+
+        const currentIndex = getLyricIndex(getCurrentPlayTime(), lyrics);
+        updateAllLyricLines(currentIndex, savedLyrics);
+    }
+
+    function renderStaticLyrics(lyrics) {
+        const itemsContainer = view.querySelector('.dynamicLyricsContainer');
+        if (itemsContainer) {
+            const html = lyrics.reduce(staticLyricHtmlReducer, '');
+            itemsContainer.innerHTML = html;
+        }
+    }
+
+    function updateLyrics(lyrics) {
+        savedLyrics = lyrics;
+
+        isDynamicLyric = Object.prototype.hasOwnProperty.call(lyrics[0], 'Start');
+
+        if (isDynamicLyric) {
+            renderDynamicLyrics(savedLyrics);
+        } else {
+            renderStaticLyrics(savedLyrics);
+        }
+    }
+
+    function getLyrics(serverId, itemId) {
+        const apiClient = ServerConnections.getApiClient(serverId);
+
+        return apiClient.ajax({
+            url: apiClient.getUrl('Audio/' + itemId + '/Lyrics'),
+            type: 'GET',
+            dataType: 'json'
+        }).then((response) => {
+            if (!response.Lyrics) {
+                throw new Error();
+            }
+            return response.Lyrics;
+        });
+    }
+
+    function bindToPlayer(player) {
+        if (player === currentPlayer) {
+            return;
+        }
+
+        releaseCurrentPlayer();
+
+        currentPlayer = player;
+
+        if (!player) {
+            return;
+        }
+
+        Events.on(player, 'timeupdate', onTimeUpdate);
+        Events.on(player, 'playbackstart', onPlaybackStart);
+        Events.on(player, 'playbackstop', onPlaybackStop);
+    }
+
+    function releaseCurrentPlayer() {
+        const player = currentPlayer;
+
+        if (player) {
+            Events.off(player, 'timeupdate', onTimeUpdate);
+            Events.off(player, 'playbackstart', onPlaybackStart);
+            Events.off(player, 'playbackstop', onPlaybackStop);
+            currentPlayer = null;
+        }
+    }
+
+    function onLyricClick(lyricTime) {
+        playbackManager.seek(lyricTime);
+        if (playbackManager.paused()) {
+            playbackManager.playPause(currentPlayer);
+        }
+    }
+
+    function onTimeUpdate() {
+        if (isDynamicLyric) {
+            const currentIndex = getLyricIndex(getCurrentPlayTime(), savedLyrics);
+            updateAllLyricLines(currentIndex, savedLyrics);
+        }
+    }
+
+    function onPlaybackStart(event, state) {
+        if (currentItem.Id !== state.NowPlayingItem.Id) {
+            onLoad();
+        }
+    }
+
+    function onPlaybackStop () {
+        appRouter.back();
+    }
```

Comment:
The lyrics page will be kept until there is a next item and will be closed at the end of the playing queue.
```suggestion
    function onPlaybackStop (_, state) {
        if (!state.NextMediaType) {
            appRouter.back();
        }
    }
```